app/src/main/java/com/cornellappdev/android/eateryblue/ui/screens/EateryDetailScreen.kt

```diff
@@ -121,7 +122,10 @@ fun EateryDetailScreen(
     val shimmer = rememberShimmer(ShimmerBounds.View)
     val context = LocalContext.current
     val modalBottomSheetState =
-        rememberModalBottomSheetState(initialValue = ModalBottomSheetValue.Hidden)
+        rememberModalBottomSheetState(
+            initialValue = ModalBottomSheetValue.Hidden,
+            skipHalfExpanded = true
+        )
```

Comment:
I added a `skipHalfExpanded` variable here to get rid of that weird behavior when the sheet would pull up only halfway. Not sure why that's the default.